src/host/ConsoleArguments.cpp

```diff
@@ -18,6 +18,7 @@ const std::wstring_view ConsoleArguments::FILEPATH_LEADER_PREFIX = L"\\??\\";
 const std::wstring_view ConsoleArguments::WIDTH_ARG = L"--width";
 const std::wstring_view ConsoleArguments::HEIGHT_ARG = L"--height";
 const std::wstring_view ConsoleArguments::INHERIT_CURSOR_ARG = L"--inheritcursor";
+const std::wstring_view ConsoleArguments::RESIZE_QUIRK = L"--resizeQuirk";

```

Comment:
So this arg only exists for us to be able to resize properly? How can a a non-WindowsTerminal terminal use this setting? I'm guessing we'd have to update documentation for this because anybody would be able to use this flag.

Reply:
Yea so that's something to discuss - we probably do need to document the flag in the public API. The conhost flags themselves are an undocumented implementation detail, but the flags to `CreatePseudoConsole` _are_ documented.

Basically other apps will work _okay_ without specifying this flag. If they want this in the future, they'll need to wait for us to add the flag to the public API.